evm-subql/project.yaml

```diff
@@ -1,467 +1,32 @@
-                  type: CurrencyId
-                - name: targetCurrencyAmount
-                  type: BalanceRequest
-              type: BalanceWrapper
-              isSubscription: false
-              jsonrpc: dex_getSupplyAmount
-              method: getSupplyAmount
-              section: dex
-            getTargetAmount:
-              description: Get target amount
-              params:
-                - name: supplyCurrencyId
-                  type: CurrencyId
-                - name: targetCurrencyId
-                  type: CurrencyId
-                - name: supplyCurrencyAmount
-                  type: BalanceRequest
-              type: BalanceWrapper
-              isSubscription: false
-              jsonrpc: dex_getTargetAmount
-              method: getTargetAmount
-              section: dex
-          evm:
-            call:
-              description: eth call
-              params:
-                - name: data
-                  type: CallRequest
-                - name: at
-                  type: BlockHash
-                  isHistoric: true
-                  isOptional: true
-              type: Raw
-              isSubscription: false
-              jsonrpc: evm_call
-              method: call
-              section: evm
-            estimateResources:
-              description: eth estimateResources
-              params:
-                - name: from
-                  type: H160
-                - name: unsignedExtrinsic
-                  type: Bytes
-                - name: at
-                  type: BlockHash
-                  isHistoric: true
-                  isOptional: true
-              type: EstimateResourcesResponse
-              isSubscription: false
-              jsonrpc: evm_estimateResources
-              method: estimateResources
-              section: evm
-          stakingPool:
-            getAvailableUnbonded:
-              description: Get Available Unbonded
-              params:
-                - name: account
-                  type: AccountId
-              type: BalanceInfo
-              isSubscription: false
-              jsonrpc: stakingPool_getAvailableUnbonded
-              method: getAvailableUnbonded
-              section: stakingPool
-            getLiquidStakingExchangeRate:
-              description: get liquid staking exchange rate
-              params: []
-              type: ExchangeRate
-              isSubscription: false
-              jsonrpc: stakingPool_getLiquidStakingExchangeRate
-              method: getLiquidStakingExchangeRate
-              section: stakingPool
-        instances:
-          council:
-            - generalCouncil
+  chaintypes:
+    # TODO: can generate this dynamically with `yarn build` using `@acala-network/types`
+    file: ./types.json
 dataSources:
-  - name: main
-    kind: substrate/Runtime
+  - kind: substrate/Runtime
     startBlock: 1
     mapping:
+      file: ./dist/index.js
```

Comment:
This path is error

Reply:
We need lists:
```
project-mandala.yaml
project-karura.yaml (startBlock: 1)
project-karura-1780000.yaml (startBlock: 1780000)
project-acala.yaml (startBlock: 1)
project-acala-840000.yaml (startBlock: 840000)
project-default.yaml (0.0.1 version)
```

Reply:
what error? after `yarn build` this path exist

Reply:
Ok. And update the `"@acala-network/eth-providers": "~2.4.6"`